Our Scripts/MPhys_model.py

```diff
@@ -60,8 +60,8 @@ def t_rec(z): #s recombination time
 def E_ion(z):#Hz/erg
     return 10**25.6 #10**(24.4 + math.log10(1 + z))
 
-def Cubic(x, a1, a2, a3, a4):
-    return a1*x**3 + a2*x**2 +a3*x +a4
+def squared(x, a1, a2, a3):
+    return a1*x**2 + a2*x +a3
```

Comment:
Is the squared function meant to replace the cubic function?

Reply:
yeah, we decided they were squares not cubics